### PR TITLE
Yusuke Proofreading Arcueid Day 3

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2.txt [hunk 1]
```diff
@@ -26,14 +26,14 @@ With a smile on her face, Kohaku-san dodges Akiha's question.
...
-"Aha, you mean like in WanWan Panic, right?"// ワンワンパニック maybe WoofWoof Panic if we're replacing wan with woof? 
+"Aha, you mean like in Woofwoof Panic, right?"
 I couldn't quite tell by her reaction, whether she believes me or not.
 <Page8>
 "...Well, to put it simply, it's something like that."
 "Hmmm―――I do not remember any of that though.
 Neither do you, right, Kohaku?"
 <Page9>
-"I suppose so... I'm sorry, Shiki-san but I don't think that happened."
+"I suppose so... I'm sorry Shiki-san but I don't think that happened."
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2.txt:36]: why remove this comma?

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt [hunk 2]
```diff
@@ -3,8 +3,8 @@ After having breakfast in the dining room, I go out to the lobby.
 Behind me is Hisui, carrying my bag.
 I suggested that I can carry my own bag, but she insisted on carrying it to the door at least.
 <Page2>
-...I seem to have a weak spot not only for Akiha but also for Hisui.
-Oh, well, to be more precise, I have a soft spot for Hisui.
+...I seem to have a weak spot not only for Akiha but Hisui as well.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt:6]: I seem to have a weak spot, not only for Akiha, but for Hisui as well.

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt [hunk 3]
```diff
@@ -3,8 +3,8 @@ After having breakfast in the dining room, I go out to the lobby.
...
+Oh, well, a soft spot would probably be more precise.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt:7]: accurate

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt [hunk 4]
```diff
@@ -56,7 +56,7 @@ I'm just curious as to what kind of gift it is."
 The wooden box is opened with a soft, dry sound.
 Inside there is a thin iron rod, about 15 centimeters long.
 <Page16>
-"It's... an iron rod."
+"It's...an iron rod."
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt:59]: we agreed to add spaces after ellipses

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt [hunk 5]
```diff
@@ -83,13 +83,13 @@ It only says 'Nanatsu-yoru'."
 "Eh...!"
 When I turn around, I see Hisui who was silent until now, peering at the knife from behind.
 <Page24>
-"Y-You surprised me... don't scare me like that, Hisui.
-You don't have to look over my shoulder like that if you want to see, here."
+"Y-You surprised me...don't scare me like that, Hisui.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt:86]: space after ellipses

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt [hunk 6]
```diff
@@ -83,13 +83,13 @@ It only says 'Nanatsu-yoru'."
...
+You don't have to look over my shoulder like that. If you want to see, here."
 <Page25>
 "Ah―――――――"
 <Page26>
 "P-Please excuse me.
-Um... that dagger is so beautiful, I could not resist."
+Um...that dagger is so beautiful, I could not resist."
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt:92]: space after ellipses

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt [hunk 7]
```diff
@@ -100,8 +100,8 @@ Since Hisui is so adamant about it, I'm starting to feel the same way.
 ...Yeah.
 Maybe this isn't a bad idea for a memento as it is.
 <Page29>
-"'Seven Nights'... is it? Maybe that's the name of this fruit knife?"
-I don't think many people name their knives... whatever it is, it's definitely vintage.
+"'Seven Nights'...is it? Maybe that's the name of this fruit knife?"
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt:103]: space after ellipses

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt [hunk 8]
```diff
@@ -100,8 +100,8 @@ Since Hisui is so adamant about it, I'm starting to feel the same way.
...
+I don't think many people name their knives...whatever it is, it's definitely vintage.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt:104]: space after ellipses

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt [hunk 9]
```diff
@@ -115,7 +115,7 @@ As I leave the gate of the mansion, I notice that there is a lot of commotion.
 <Page33>
 "...What is it? Isn't it kind of noisy over there?"
 "The thing is, they found some blood on the road this morning."
-"Blood...? You mean... bloodstains?"
+"Blood...? You mean...bloodstains?"
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt:118]: space after ellipses

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt [hunk 10]
```diff
@@ -125,7 +125,7 @@ It seems that while you were asleep, the police came to inquire about what happe
 "――――――"
 <Page36>
 The public road east of the mansion, ―――that's where I went to check last night.
-Bloodstains... splattered red colors clinging to the walls.
+Bloodstains...splattered red colors clinging to the walls.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2B.txt:128]: space after ellipses

**Review by Avuxo — CHANGES_REQUESTED** [review]
changing references to lines to plural is a good move. I see a lot of spaces being removed from ellipses. I wonder if this was based on a branch before Ross made the bulk change?

File: script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2.txt (hunk 1 above)
Comment (yk1512) [reply script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_2.txt:36]: I was trying to get rid of the awkward pause there. "I'm sorry Shiki-san, but I don't think that happened" might be better actually.

**Pull request merged**